symengine/printer.cpp

```diff
@@ -124,6 +124,47 @@ void StrPrinter::bvisit(const Interval &x)
     str_ = s.str();
 }
 
+void StrPrinter::bvisit(const BooleanAtom &x)
+{
+    if (x.get_val()) {
+        str_ = "True";
+    } else {
+        str_ = "False";
+    }
+}
+
+void StrPrinter::bvisit(const Contains &x)
+{
+    std::ostringstream s;
+    s << "Contains(" << apply(x.get_expr()) << ", " << apply(x.get_set())
+      << ")";
+    // s << apply(x.get_expr()) << " " << "∈" << " " << apply(x.get_set());
```

Comment:
This might become obscure for

```
auto int1 = interval(integer(1), integer(2), false, false);
auto p = contains(sqrt(i2), int1);
std::cout << *p;
```